stories/ui/IconPromotedBadge.js

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import IconPromotedBadge from 'ui/components/IconPromotedBadge';
+
+// TODO: remove the comment below once
+// https://github.com/yannickcr/eslint-plugin-react/issues/2298 is fixed.
```

Comment:
the issue is closed, so it's fixed?

Reply:
I checked and the issue here does not seem to be fixed. I.e., we still need the `eslint` comment. I'm not sure why, but it's not something I think is worth addressing for this PR.